apps/bare-expo/android/app/src/main/java/dev/expo/payments/MainApplication.java

```diff
@@ -10,6 +10,7 @@
 import com.facebook.react.config.ReactFeatureFlags;
 import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint;
 import com.facebook.react.defaults.DefaultReactNativeHost;
+import com.facebook.react.flipper.ReactNativeFlipper;
```

Comment:
nit: do you want to migrate MainApplication / MainActivity to kotlin as 0.73?

Reply:
Sure, I can migrate this as the part 2 of this PR

Reply:
will be addressed here https://github.com/expo/expo/pull/25453